tasks/install_redhat.yml

```diff
@@ -18,20 +18,20 @@
 
       https://github.com/r-pufky/ansible_pihole/issues/10
 
-- name: 'redhat | install dependencies'
+- name: 'Redhat | install dependencies'
   ansible.builtin.dnf:
     name:  '{{ pihole_default_packages_rpm }}'
     state: 'latest'
     update_cache: true
 
-- name: 'redhat | set SELinux to permissive'
+- name: 'Redhat | set SELinux to permissive'
```

Comment:
For these changes (capping a letter), this is what my initial comment before review was about. Please leave these as lowercase to be consistent with the rest of the role.

I don't think there is any set-in-stone ansible guidance here other than personal preference for the roles I've created (all lowercase unless an acronym, etc; which makes it easier to read instead of Camel Case All The Time.

```TASK [pihole : redhat | set SELinux to permissive]```

vs.

```TASK [pihole : Redhat | set SELinux to permissive]```

Reply:
There is ansible-lint guidance...
name[casing]: https://ansible.readthedocs.io/projects/lint/rules/name/

Reply:
If an outsider's suggestion is welcome, I'd take this a step further across the whole role. Casing is best-practice, per ansible-lint, but if you're going to do that it might make more sense to keep the casing consistent with ansible facts (when appropriate). By that I mean when you import the install_redhat.yml in the install.yml, you reference the casing as ansible defines the os_family:

` when: ansible_os_family == 'RedHat'`

If the case change is worth doing at all across the role for consistency, I'd push for this to be 'RedHat' everywhere it can be.

Reply:
agreed but ultimately, this PR is really about the CI and ensure it is green. I would put the extra work in different PR(s) as really about chore/style type of change.

current red parts of pipeline
* default-bare on 20.04 with no logs...
* default (molecule/docker): pihole install script failing on ubuntu, redhat failing because rockylinux unsupportedn fedora37 same than ubuntu.

Reply:
> If an outsider's suggestion is welcome, I'd take this a step further across the whole role. Casing is best-practice, per ansible-lint, but if you're going to do that it might make more sense to keep the casing consistent with ansible facts (when appropriate). By that I mean when importing the install_redhat.yml in the install.yml, it references the casing as ansible defines the os_family:
> 
> ` when: ansible_os_family == 'RedHat'`
> 
> If the case change is worth doing at all across the role for consistency, I'd push for this to be 'RedHat' everywhere it can be.

Pulled out into a separate bug.